perftracker_django/urls.py

```diff
@@ -38,6 +38,10 @@
         views.pt_comparison_tables_info_json),
     url(r'^api/v(?P<api_ver>\d+.\d+)/(?P<project_id>\d+)/comparison/(?P<cmp_id>\d+)/tables$',
         views.pt_comparison_tables_info_json),
+    url(r'^api/v(?P<api_ver>\d+.\d+)/(?P<project_id>\d+)/comparison/(?P<cmp_id>\d+)/analyze$',
```

Comment:
may be it is time to stop forming/using wrong URIs
1) (minor) api_ver can handled not per uri but some common root (I bet Django has support for that)
2) don't pass/don't parse unused parameters ( e.g. project_id) - only those that matter. For pt_comparison_section_properties_save none matters, but may be it is reasonable to keep cmp_id + section_id just fore reference.